pkgs/os-specific/linux/trace-cmd/default.nix

```diff
@@ -1,25 +1,39 @@
 { stdenv, fetchgit, asciidoc, docbook_xsl, libxslt }:
-
-stdenv.mkDerivation rec {
+let
+  srcSpec = import ./src.nix;
+  shortRev = builtins.substring 0 7 srcSpec.rev;
+in stdenv.mkDerivation rec {
   name    = "trace-cmd-${version}";
-  version = "2.6";
+  version = "2.8-${shortRev}";
 
-  src = fetchgit {
-    url    = "git://git.kernel.org/pub/scm/linux/kernel/git/rostedt/trace-cmd.git";
-    rev    = "refs/tags/trace-cmd-v${version}";
-    sha256 = "15d6b7l766h2mamqgphx6l6a33b1zn0yar2h7i6b24ph6kz3idxn";
-  };
+  src = fetchgit srcSpec;
+
+  patches = [ ./fix-Makefiles.patch ];
 
   buildInputs = [ asciidoc libxslt ];
```

Comment:
If these are only used for building documentation, they should go to `nativeBuildInputs`.

Reply:
Good point. Will fix.